subsys/mpsl/cx/Kconfig

```diff
@@ -21,13 +21,16 @@ config MPSL_CX_NRF700X_SUPPORT
 config MPSL_CX
 	bool "Radio Coexistence interface support"
 	depends on MPSL_CX_ANY_SUPPORT
+	depends on !MPSL_FEM && !SOC_SERIES_NRF53X
```

Comment:
Is this correct? I'm assuming the intention was to express that
```
config MPSL_CX
    depends on !(MPSL_FEM && SOC_SERIES_NRF53X) # we don't want CX && FEM && NRF53 at the same time
```

which without parentheses should be:
```
config MPSL_CX
    depends on !MPSL_FEM || !SOC_SERIES_NRF53X
```
In my understanding in its current shape, this change removes support for CX on nRF53 _and_ it removes support for CX + FEM for all families.

Reply:
You are right, good catch, thank you.